driver/main.c

```diff
@@ -955,24 +939,12 @@ static long ppm_ioctl(struct file *filp, unsigned int cmd, unsigned long arg)
 	switch (cmd) {
 	case PPM_IOCTL_DISABLE_DROPPING_MODE:
 	{
-		struct event_data_t event_data;
```

Comment:
Avoid sending a `PPME_SCAPEVENT_E` when dropping mode is disabled.

Reply:
This isn't completely dumb (lets you tell when in the event stream the ioctl happened) but I think we can drop it